lib/fluent/plugin/monitoring.rb

```diff
@@ -16,7 +16,7 @@ module Monitoring
   # Base class for the monitoring registry.
   class BaseMonitoringRegistry
     def counter(_name, _desc)
-      _undefined
```

Comment:
This was intentional, I think, to prevent you from creating a counter for an undefined registry. See my suggestion for an alternative approach.